disabled_apps/clock/manifest.webmanifest

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Clock",
+  "description": "Gaia Clock",
+  "lang": "en",
+  "short_name": "Clock",
+  "scope": "./",
```

Comment:
I'm not sure this is a valid scope. If the Clock app is hosted at "http://mozilla-b2g.github.io/gaia/dead_apps/clock/" then the scope would be "/gaia/dead_apps/clock/"

Or you could leave the scope property out